src/index.jsx

```diff
@@ -39,6 +39,7 @@ initialize({
         FEATURE_LEARNER_CREDIT_MANAGEMENT: process.env.FEATURE_LEARNER_CREDIT_MANAGEMENT || hasFeatureFlagEnabled('LEARNER_CREDIT_MANAGEMENT') || null,
         FEATURE_CONTENT_HIGHLIGHTS: process.env.FEATURE_CONTENT_HIGHLIGHTS || hasFeatureFlagEnabled('CONTENT_HIGHLIGHTS') || null,
         ENTERPRISE_SUPPORT_PROGRAM_OPTIMIZATION_URL: process.env.ENTERPRISE_SUPPORT_PROGRAM_OPTIMIZATION_URL || null,
+        ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL: process.env.ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL || null,
```

Comment:
`ENTERPRISE_SUPPORT_LEARNER_CREDIT_URL` is no longer coming in as an environment variable if its not defined in `.env.development` or the frontend-specific edx-internal config, so I believe this line will always evaluate to `null`. It might be worth removing it now?

Reply:
As discussed, we are leaving a value defined in .env.development for now, at least until that file is fully deprecated and a replacement provided.  This helps not completely break a misconfigured (default) local development environment that hasn't enabled the config API.